### PR TITLE
HTML Email template for Totango and beyond!

### PR DESCRIPTION
**Pull request by chadbercea**
This commit covers the base starting point for html email tempaltes. This is ready to work wtih Totango.

### CODE REVIEW

**Comment by chadbercea** [comment]
@tfoxnc this bad boy is ready for you to review!

File: Product Customer Newsletter Email Template/email-basic-template.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                                        <tr>
+                                            <p style="font-family: sans-serif; font-size: 22px; font-weight: bold; color: #1c95b4; margin-top: 25px; margin-bottom: 25px; text-align: center"; >Thank you for checking out this template!</p>
+                                        </tr>
+                                        <tr>
+                                            <td style="font-family: sans-serif; font-size: 16px; vertical-align: top; padding: 0 40px;">
+                                                <p style="font-fa mily: sans-serif; font-size: 16px; font-weight: normal; margin-bottom: 15px;">
+                                                    <b>Hi</b>
+                                                    <p>Design enablement put together a template for the CS team to be deployed via Totango and the last time I checked, they still didn't have a way to properly store email templates.</p>
+                                                    <p>With that said, would you like to keep this tempalte in Github, or would you like the code pasted in a Confluence page ready to copy and paste?</p>
```
Comment (tfoxnc) [inline Product Customer Newsletter Email Template/email-basic-template.html:31]:
```suggestion
                                                    <p>With that said, would you like to keep this template in Github, or would you like the code pasted in a Confluence page ready to copy and paste?</p>
```

File: Product Customer Newsletter Email Template/email-basic-template.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Email Title</title>
```
Comment (tfoxnc) [inline Product Customer Newsletter Email Template/email-basic-template.html:6]:
```suggestion
        <title>Email title</title>
```

Comment (tfoxnc) [reply Product Customer Newsletter Email Template/email-basic-template.html:6]: Our style guide says to use sentence capitalization for titles.

**Review by tfoxnc — CHANGES_REQUESTED** [review]

**Comment by tfoxnc** [comment]
I would rather keep it in GitHub since it is code and easier to suggest code changes in GitHub.

**Pull request closed without merging**